recipes/biogeme-optimization/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "biogeme-optimization" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/biogeme_optimization-{{ version }}.tar.gz
+  sha256: c202fb1b5eaecffdca04f98201ac39cc813dee2c3bb4c98cf079a868bda1c6bb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy >=1.24.2
+    - scipy >=1.10.1
+    - matplotlib-base >=3.8.2
+
+test:
+  imports:
+    - biogeme_optimization
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://biogeme.epfl.ch
+  summary: Various optimization algorithms for teaching and research
+  license: Custom
+
```

Comment:
The license files are included in the tarballs, so you still need the license file name.